iroh-net/src/hp/derp/clients.rs

```diff
@@ -63,19 +65,45 @@ impl Client {
     }
 
     pub fn send_packet(&self, packet: Packet) -> Result<(), SendError> {
-        try_send(&self.conn.client_channels.send_queue, packet)
+        let res = try_send(&self.conn.client_channels.send_queue, packet);
+        if res.is_ok() {
+            inc!(Metrics, send_packets_sent);
+        }
+        res
     }
 
     pub fn send_disco_packet(&self, packet: Packet) -> Result<(), SendError> {
-        try_send(&self.conn.client_channels.disco_send_queue, packet)
+        let res = try_send(&self.conn.client_channels.disco_send_queue, packet);
+        if res.is_ok() {
+            inc!(Metrics, disco_packets_sent);
+        }
```

Comment:
why not mark as `dropped`

Reply:
left a comment explaining that this is not the last time we attempt to send the packet, so it isn't considered "dropped" yet